flume/pom.xml

```diff
@@ -0,0 +1,295 @@
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>malhar</artifactId>
+    <groupId>org.apache.apex</groupId>
+    <version>3.8.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>malhar-flume</artifactId>
+  <packaging>jar</packaging>
+  <name>Apache Apex Malhar Flume Support</name>
+
+  <profiles>
+    <profile>
+      <id>release</id>
+      <properties>
+        <package.username>flume</package.username>
+        <rpm.skip>package</rpm.skip>
+        <rpm.phase>${rpm.skip}</rpm.phase>
+      </properties>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>rpm-maven-plugin</artifactId>
+            <version>2.1-alpha-4</version>
+            <executions>
+              <execution>
+                <phase>${rpm.phase}</phase>
+                <id>generate-sink-rpm</id>
+                <goals>
+                  <goal>attached-rpm</goal>
+                </goals>
+                <configuration>
+                  <license>Apache License, Version 2.0</license>
+                  <version>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}.${parsedVersion.incrementalVersion}</version>
+                  <release>${parsedVersion.qualifier}${parsedVersion.buildNumber}</release>
+                  <workarea>target/sink-rpm</workarea>
+                  <classifier>sink</classifier>
+                  <name>apex-malhar-flume-sink</name>
+                  <distribution>Apache Apex Malhar ${project.version}</distribution>
+                  <group>Messaging Client Support</group>
+                  <icon>src/main/resources/logo.gif</icon>
+                  <packager>Apex Build System</packager>
+                  <prefix>${package.prefix}</prefix>
+                  <changelogFile>src/changelog</changelogFile>
+                  <defineStatements>
+                    <defineStatement>_unpackaged_files_terminate_build 0</defineStatement>
+                  </defineStatements>
+                  <mappings>
+                    <mapping>
+                      <directory>${package.prefix}/flume-${project.version}/lib</directory>
+                      <filemode>750</filemode>
+                      <username>${package.username}</username>
+                      <groupname>${package.groupname}</groupname>
+                      <artifact></artifact>
+                      <dependency>
+                        <includes>
+                          <include>org.apache.apex:apex-api:jar:${apex.core.version}</include>
+                          <include>com.datatorrent:dt-netlet:jar:1.2.0</include>
```

Comment:
Is it necessary to repeat the version number here? This will break when we update core.